lib/drivers/dexcomDriver.js

```diff
@@ -645,6 +645,7 @@ module.exports = function (config) {
       }
       if (datum.annotation) {
         annotate.annotateEvent(cbg, datum.annotation);
+        cbg._forceUpdate = true;
```

Comment:
and this is the flag we add to say this must be updated / or we can force an update?